package.json

```
@@ -17,12 +17,18 @@
     "test:dev": "jest --watch",
     "test:e2e": "jest --config e2e/jest-e2e.json",
     "prerelease": "npm run build",
-    "release": "release-it"
+    "release": "release-it",
+    "---execution---": "",
```

```suggestion
    "---execution---": "",
```
should we rename it to manual-test?

---

do you mean the section ("---execution---") or section & scripts (start -> 'manual-test')?

---

@flamewow updated code suggestion